industrial_ci/src/ros.sh

```diff
@@ -88,6 +88,10 @@ function _set_ros_defaults {
         _ros2_defaults "jammy"
         ;;
     "false")
+        export BUILDER=${BUILDER:-colcon}
+        export ROS_VERSION=0
+        export ROS_VERSION_EOL=false
+        export ROS_PYTHON_VERSION=${ROS_PYTHON_VERSION:-3}
```

Comment:
Those are questionable defaults.

Reply:
What are your suggested alternatives? Python2 is EOL, so Python3 seems reasonable. `colcon` is the only builder I am familar with that can build both ROS1 and ROS2 code. The ROS version cannot be EOL since this is for a ROS-independent build. `ROS_VERSION` is required by other parts of ICI, so anything < 1 seems okay